index.md

```diff
@@ -57,10 +44,11 @@ page](new_for_03.html).
 * [Git](git.html)
 * [Extensions](extensions.html)
 * [Polyglot IDE](polyglot.html)
+* [Remote Execution](remote-execution.html)
 
 ## Video feature walk-throughs
 
-Here's the video for 0.4:
+Here's the most recent walk-through video for RTVS 0.4:
```

Comment:
That's the latest video. Pretty sure it was for 0.4? If not, happy to bump it to 0.5 :)